core/server/web/middleware/admin-redirects.js

```diff
@@ -1,11 +1,7 @@
-const express = require('express'),
-    urlService = require('../../services/url');
+const express = require('express');
+const urlService = require('../../services/url');
 
-const adminRedirect = function adminRedirect(path) {
-    return function doRedirect(req, res) {
-        return urlService.utils.redirectToAdmin(301, res, path);
-    };
-};
+const adminRedirect = path => (req, res) => urlService.utils.redirectToAdmin(301, res, path);
```

Comment:
IMO very hard to read. What do you think

Reply:
Agreed :confused: It's actually readable once the eye gets used to arrow functions everywhere, but still need to think twice once you see a function return like this...

Maybe this way it'd be more readable?:
```javascript
const adminRedirect = (path) =>  {
     return (req, res) => urlService.utils.redirectToAdmin(301, res, path);
}
```

Reply:
Yeah, I'd vote for not using this pattern of arrow function which does implicit `return` and doesn't use brackets, and instead only use the style which has both params bracket -`()`- and function wrappers - `{}`. Main reason being readability, plus in case of multiline functions we'll have to fallback to this so it stays consistent in how we use arrow functions.

Reply:
Cool agree 👍